developer_guide/installation/installation_workstation.rst

```diff
@@ -375,6 +375,33 @@ Adding the vhost name
     $ sudo gedit /etc/hosts
     127.0.0.1    akeneo-pim.local
 
+Configuring tasks via crontab
+*****************************
+
+The application needs the following tasks to be executed in background on a regular basis :
+
+.. code-block:: bash
+    :linenos:
+
+    # for CE & EE
+    /path/to/php /path/to/pim/root/app/console pim:completeness:calculate
```

Comment:
is '/path/to' really necessary ? I mean we "get" it right ?  I mean it doesn't help to read that's all.